### DISCUSSION TITLE
The "type_map.raw" in the data set generated by 02.fp does not match

### DISCUSSION THREAD

**Discussion by Qjump** [deepmodeling/deepmd-kit#2583] [Q&A]
In my param.json file, the type_map is set to "C Cu H N O S", and the initial data set's type_map.raw is also "C Cu H N O S". Due to the presence of multiple systems, if only C, Cu, H, and O elements are present, in my initial data set, type_map.raw is "C Cu H N O S" and the corresponding type.raw is "0 1 2 4". However, after training through iterations, the type_map.raw in the generated 02.fp data set is "C Cu H O", and the corresponding type.raw is "0123". How can I modify the settings to make it consistent with the type_map.raw of the initial data set?
My dpgen version is 0.11.1, and my dpdata version is 0.2.14.

**Comment by njzjz** [comment]
Actually, neither dpdata nor deepmd-kit requires that type_map.raw is consistent.
